### ISSUE TITLE
Installing it locally results in TableRex error

### ISSUE THREAD

**Issue by robinjoseph08** [unnawut/licensir#13]
I used `mix archive.install hex licensir 0.5.0` to install `licensir` locally, and while that completed successfully, when I try to run `mix licenses` in an Elixir project, I get the following error:

```
** (UndefinedFunctionError) function TableRex.quick_render!/2 is undefined (module TableRex is not available)
    TableRex.quick_render!([["artificery", nil, "Apache 2.0"], ["bunt", nil, "Unsure (found: MIT, Unrecognized license file content)"], ["certifi", nil, "BSD"], ["combine", nil, "MIT"], ["confex", nil, "Unsure (found: LISENSE.md, MIT)"], ["connection", nil, "Apache 2.0"], ["cowboy", nil, "ISC"], ["cowlib", nil, "ISC"], ["credo", nil, "Unsure (found: MIT, Unrecognized license file content)"], ["crontab", nil, "Unsure (found: MIT, Unrecognized license file content)"], ["db_connection", nil, "Apache 2.0"], ["decimal", nil, "Apache 2.0"], ["dialyxir", nil, "Apache 2.0"], ["distillery", nil, "MIT"], ["double_metaphone", nil, "MIT"], ["ecto", nil, "Apache 2.0"], ["ecto_sql", nil, "Apache 2.0"], ["ex_aws", nil, "MIT"], ["ex_aws_kinesis", nil, "MIT"], ["ex_aws_s3", nil, "MIT"], ["excoveralls", nil, "Unsure (found: MIT, Unrecognized license file content)"], ["exjsx", nil, "Unsure (found: MIT, Unrecognized license file content)"], ["gen_stage", nil, "Apache 2.0"], ["gen_state_machine", nil, "Apache 2.0"], ["gettext", nil, "Apache 2.0"], ["hackney", nil, "Apache 2.0"], ["httpoison", nil, "Unsure (found: MIT, Unrecognized license file content)"], ["idna", nil, "Unsure (found: BSD, MIT)"], ["jason", nil, "Apache 2.0"], ["jsx", nil, "MIT"], ["lazy_agent", nil, "MIT"], ["libring", nil, "MIT"], ["litmus", nil, "MIT"], ["logger_elixir", nil, "MIT"], ["metrics", nil, "BSD"], ["metrics_elixir", nil, "MIT"], ["mime", nil, "Apache 2.0"], ["mimerl", nil, "MIT"], ["parse_trans", nil, "Apache 2.0"], ["plug", nil, "Apache 2.0"], ["poison", nil, "CC0-1.0"], ["postgrex", nil, "Apache 2.0"], ["quantum", nil, "Unsure (found: Apache License 2.0, Apache 2.0)"], ["ranch", nil, "ISC"], ["redix", nil, "MIT"], ["remodel", nil, "Apache 2.0"], ["sentry", nil, "MIT"], ["simetric", nil, ...], ["ssl_verify_fun", ...], [...], ...], ["Package", "Version", "License"])
    lib/mix/tasks/licenses.ex:23: Mix.Tasks.Licenses.run/1
    (mix) lib/mix/task.ex:331: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:79: Mix.CLI.run_task/2
```

I'm using the following versions:

```
Erlang/OTP 22 [erts-10.4.4] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.9.1 (compiled with Erlang/OTP 22)
```

Thanks!

**Comment by unnawut** [comment]
That's definitely to do with me having added TableRex to 0.5.0. I'll have a look tonight. Thanks for reporting!

**Comment by unnawut** [comment]
Fixed in https://github.com/unnawut/licensir/commit/8cabbadbfdbd7de6b9c120ad9c7954602b100806 and released as [v0.5.1](https://github.com/unnawut/licensir/releases/tag/v0.5.1). Should work now :)

**Comment by doughsay** [comment]
Can confirm it works!
